frontend/pubspec.yaml

```diff
@@ -22,18 +22,18 @@ dependencies:
       ref: git-release-0.18.0
   provider: ^6.0.3
   graphql_flutter: ^5.1.2
-  url_launcher: ^6.1.6
+  url_launcher: ^6.2.5
   flutter_svg: ^2.0.2
-  permission_handler: ^10.2.0
+  permission_handler: ^11.3.0
   package_info_plus: ^4.0.1 # for about dialog
-  mobile_scanner: 3.1.1 # Pinning this version because of https://github.com/juliansteenbakker/mobile_scanner/issues/582
+  mobile_scanner: ^4.0.1 # Pinning this version because of https://github.com/juliansteenbakker/mobile_scanner/issues/582
```

Comment:
this has to be tested properly. in the issue it was mentioned that it should be fixed with XCode 15 and i updated the sdk version in another pr, so we should be fine